types/react-select/src/Async.d.ts

```diff
@@ -5,43 +5,50 @@ import manageState from './stateManager';
 import { GroupedOptionsType, OptionsType, InputActionMeta, OptionTypeBase } from './types';
 
 export interface AsyncProps<OptionType extends OptionTypeBase> {
-  /* The default set of options to show before the user starts searching. When
-     set to `true`, the results for loadOptions('') will be autoloaded.
-     Default: false. */
-  defaultOptions?: GroupedOptionsType<OptionType> | OptionsType<OptionType> | boolean;
-  /* Function that returns a promise, which is the set of options to be used
-     once the promise resolves. */
-  loadOptions: (inputValue: string, callback: ((options: OptionsType<OptionType>) => void)) => Promise<any> | void;
-  /* If cacheOptions is truthy, then the loaded data will be cached. The cache
-     will remain until `cacheOptions` changes value.
-     Default: false. */
-  cacheOptions?: any;
+    /* The default set of options to show before the user starts searching. When
+       set to `true`, the results for loadOptions('') will be autoloaded.
+       Default: false. */
+    defaultOptions?: GroupedOptionsType<OptionType> | OptionsType<OptionType> | boolean;
```

Comment:
```suggestion
    /**
     * The default set of options to show before the user starts searching. When
     * set to `true`, the results for loadOptions('') will be automatically loaded.
     *
     * @default false
     */
    defaultOptions?: GroupedOptionsType<OptionType> | OptionsType<OptionType> | boolean;
```

Before you groan, hear me out! With formatting comments _exactly_ like [this](https://jsdoc.app/), VS Code, Sublime, and JetBrains have an automatic or (often-installed) in-editor integration. Additionally, tools like react-docgen (used by Storybook) can automatically parse the comments and turn them into documentation with their Docs mode.

I'd _love_ if you made this change while here... or at least let's add it to the road map of the TS conversion ticket. I don't making that change. It's so trivial too... Would be a good one to tweet out maybe?!

Reply:
I'll do it in a follow-up PR. ;)

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50837